### ISSUE TITLE
[FTR] Delete Story in Story Archive

### ISSUE THREAD

**Issue by xbvrhlmb** [austinhuang0131/barinsta#1297]
We can join Settings -> Story archive
but we not can delete story in Story Archive

make function button delete this stories
and i think very nice have button "delete all archive story in this date" 
Example i have in 01:01:00  30 stories, 
in order not to delete one at a time I can delete all at once)

**Comment by Martin5001** [comment]
Yeah, having the ability to delete multiple archived stories at once would be really great. Something I miss in the official app.
